packages/eslint-config/README.md

```diff
@@ -0,0 +1,28 @@
+# @talend/eslint-config
+
+This module is a set of shared eslint configuration.
+
+## How to use
+
+Create an `.eslintrc` file in the folder you need to lint.
+
+```json
+{
+  "extends": "@talend/eslint-config"
```

Comment:
That would require an `npm install` too, right ?

Reply:
I add it